android/src/main/java/io/radar/react/RNRadarModule.java

```diff
@@ -242,6 +304,22 @@ public void rejectEvent(String eventId) {
         Radar.rejectEvent(eventId);
     }
 
+    @ReactMethod
+    public void startTrip(ReadableMap optionsMap) {
+        try {
+            JSONObject optionsObj = RNRadarUtils.jsonForMap(optionsMap);
+            RadarTripOptions options = RadarTripOptions.fromJson(optionsObj);
+            Radar.startTrip(options);
+        } catch (JSONException e) {
+
```

Comment:
report the error back to the caller here?

Reply:
`startTrip()` doesn't accept a callback, but let me add some error logging for exceptions like we have in the Cordova lib.